src/components/Alert/Alert.tsx

```diff
@@ -43,7 +43,8 @@ export const Alert: FunctionComponent<AlertProps> = ({
 						<Spinner light={dark} />
 					)}
 				</Spacer>
-				{message || children}
+				{!!message && <span dangerouslySetInnerHTML={{ __html: message.toString() }} />}
+				{!!children && children}
```

Comment:
I think this can just be `{children}` because if the value is `null`, it will not render anything anyway.